src/module-config/module-config.ts

```diff
@@ -38,7 +19,26 @@ export function provide(config) {
   configs.push(config);
 }
 
-export function getConfig(moduleName) {
+let importMapConfigHasBeenAdded = false;
+export async function getConfig(moduleName: string): Promise<Object> {
```

Comment:
I am pretty sure that it should be `Promise<object>` instead of `Promise<Object>`. See https://mariusschulz.com/blog/the-object-type-in-typescript#object-vs-object-vs.

`Object` refers to the Object prototype, whereas `object` refers to a primitive object.